AzFunction/BroomBotUtils.cs

```diff
@@ -0,0 +1,152 @@
+using System.Collections.Generic;
+using Microsoft.TeamFoundation.SourceControl.WebApi;
+using Microsoft.TeamFoundation.Core.WebApi;
+using System.Threading.Tasks;
+using System.Linq;
+using System;
+using Microsoft.VisualStudio.Services.WebApi.Patch.Json;
+using Microsoft.VisualStudio.Services.WebApi.Patch;
+namespace BroomBot
+{
+    public static class BroomBotUtils
+    {
+        public static async Task<IList<GitPullRequest>> GetPullRequests(
+            GitHttpClient gitClient,
+            string project)
+        {
+            List<GitPullRequest> pullCollection = new List<GitPullRequest>();
+            List<GitRepository> repos = await gitClient.GetRepositoriesAsync(project);
+            GitPullRequestSearchCriteria searchCriteria = new GitPullRequestSearchCriteria
+            {
+                Status = PullRequestStatus.Active
+            };
+
+            foreach (GitRepository repo in repos)
+            {
+                var pulls = await gitClient.GetPullRequestsAsync(project, repo.Id, searchCriteria);
+
+                foreach (var pull in pulls)
+                {
+                    pullCollection.Add(pull);
+                }
+            }
+
+            return pullCollection;
+        }
+
+        public static async Task<Dictionary<GitPullRequest, bool>> CheckPullRequestFreshness(
+            GitHttpClient gitClient,
+            string project,
+            IList<GitPullRequest> pullRequests,
+            DateTime staleDate,
+            string botId)
+        {
+            Dictionary<GitPullRequest, bool> pullCollection = new Dictionary<GitPullRequest, bool>();
```

Comment:
if we decide to make more properties instead of just freshness boolean - then we might think about creating a custom object for GitPullRequest with a few additional properties. But for now this looks good

Reply:
Agreed. Right now the only one I thought of as being relevant was whether or not there's been an update since the last bot comment. If we think of more, we can add more 😁